pkg/backend/openshiftcluster.go

```diff
@@ -38,22 +38,36 @@ func (ocb *openShiftClusterBackend) try(ctx context.Context) (bool, error) {
 
 	log.Print("dequeued")
 	atomic.AddInt32(&ocb.workers, 1)
+	ocb.m.EmitGauge("backend.workers.count.openshiftcluster", 1, map[string]string{})
+
 	go func() {
 		defer recover.Panic(log)
 
+		t := time.Now()
+
+		// metrics and worker counter decrease logic shares defer
```

Comment:
try to avoid comments of the form
```
// do thing
doThing()
```

either (preferably) make the behaviour clear enough that the comment is not needed or make the comment add insight